packages/pds/src/account-manager/scope-decoder.ts

```diff
@@ -0,0 +1,46 @@
+import Redis from 'ioredis'
+import { Agent } from '@atproto/api'
+import { DAY } from '@atproto/common'
+import { OAuthScope } from '@atproto/oauth-provider'
+import { CachedGetter } from '@atproto-labs/simple-store'
+import { SimpleStoreMemory } from '@atproto-labs/simple-store-memory'
+import { SimpleStoreRedis } from '@atproto-labs/simple-store-redis'
+
+type EncodedScope = `enc:${string}`
```

Comment:
This is a bit of a nit and I am not sure whether it would be worth changing the terminology everywhere, but I think of this as "referencing" and "dereferencing" the scope rather than "encoding" and "decoding" it.  I wonder if @DavidBuchanan314 may have thoughts on the terminology too.

Reply:
I must admit, the encode/decode terminology threw me a little, it took me a while to figure out which feature was being implemented here. But I can't think of anything much better!

One thought is "compress"/"decompress". Also a little misleading since we're not using compression per se, but the underlying motivation is in reducing size.

Specifically for this string prefix, I think `ref:` would work best regardless of terminology elsewhere - "enc" makes me think of encryption.